src/tests/nativeaot/SmokeTests/ComWrappers/ComWrappers.csproj

```diff
@@ -1,13 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <CLRTestTargetUnsupported Condition="'$(TargetsWindows)' != 'true'">true</CLRTestTargetUnsupported>
-
-    <!-- Shouldn't need this: https://github.com/dotnet/linker/issues/2618 -->
-    <NoWarn>$(NoWarn);IL2050</NoWarn>
-
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```

Comment:
For tests that are legitimately marked as `RequiresProcessIsolation` due to something in the test, we should add comments explaining why so when we come around and clean up some of them in the future, we know which ones need to stay.
```suggestion
    <!-- Registers a global ComWrappers instance for marshalling -->
    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```